tests/contrib/epidemiology/test_models.py

```diff
@@ -495,6 +496,56 @@ def test_regional_smoke(duration, forecast, options, algo):
     assert samples["I"].shape == (num_samples, duration + forecast, num_regions)
 
 
+class RegionalSIRModelWithFinalize(RegionalSIRModel):
+    def finalize(self, params, state):
+        I = state["I"]
+        I_mean = I.mean(dim=[-1, -2], keepdim=True).expand_as(I)
+        with self.region_plate, self.time_plate:
```

Comment:
Can you explain the semantics of the `plate`s here since things are no longer independent?

Reply:
I'm simply using plates for convenience, to avoid a few `.unsqueeze(-2)`s.